salt/personalised-covers/aws.sls

```diff
@@ -0,0 +1,10 @@
+aws-credentials:
+    file.managed:
+        - name: /home/{{ pillar.elife.deploy_user.username }}/.aws/credentials
+        - source: salt://personalised-covers/config/home-user-.aws-credentials
+        - user: {{ pillar.elife.deploy_user.username }}
```

Comment:
are the PHP processes running in php-fpm use this? If so, we may need to give them to the `pillar.elife.webserver.username` too